File: Administrativo/spring-erp-web/SpringNg/src/app/erp_module/programasocial/consumoplantilla/dominio/DtoConsumoPlantilla.ts
import { ParametroPaginacionGenerico } from "src/app/erp_module/shared/dominio/ParametroPaginacionGenerico";

export class DtoConsumoPlantilla {

  codInstitucion: string;
  codItem: string;
  nomItem: string;
  nomUnidad: string;

  paginacion: ParametroPaginacionGenerico;

    constructor() {
        this.paginacion = new ParametroPaginacionGenerico();
    }
  
}

File: Administrativo/spring-erp-servicio/net/royal/spring/programasocial/dominio/dtos/DtoConsumoPlantilla.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace net.royal.spring.rrhh.dominio.dto
{
    public class DtoConsumoPlantilla
    {
        public String codInstitucion { get; set; }
        public String codItem { get; set; }

        public Nullable<Int32>  plantilla { get; set; }
        public String nomItem{ get; set; }
        public String nomUnidad { get; set; }
     

    }
}
